src/form.svelte

```diff
@@ -12,13 +12,36 @@
   let saveState = "";
   let errorMessage = "";
   let availableTagNames = []
+  let formTitle = "";
 
   async function init() {
     const tabInfo = await getCurrentTabInfo();
     url = tabInfo.url;
     titlePlaceholder = tabInfo.title;
     const availableTags = await getTags().catch(() => [])
     availableTagNames = availableTags.map(tag => tag.name)
+
+   initCurrentBookmarkData();
+  }
+
+  async function initCurrentBookmarkData() {
+    formTitle = 'Searching for existing Bookmarks';
+
+     var bookmarkSearch = await lookForCurrentBookmark(url)
+        .catch(() => []);
+
+    if ( bookmarkSearch && bookmarkSearch.length > 0 ) {
```

Comment:
```suggestion
    if (bookmarkSearch && bookmarkSearch.length > 0) {
```
Let's align the code style with the rest of code.